index.js

```diff
@@ -26,24 +28,26 @@ $.ajax({
     type: "GET"
 });
 
-
-function fillingOfVoids(obj) {
-    if (obj) {
-        return obj;
-    } else {
-        obj = 'Unknown';
-        return obj;
+class Book {
+    constructor(src) {
+        this.data = [src.authors, src.title, src.categories, src.publisher, src.publishedDate];
```

Comment:
Don't use this structure. Just change to ` this.data = src` and work with object.